devel/mold/Portfile

```
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+version             1.5.1
+
+github.setup        rui314 mold v1.5.1
+github.tarball_from archive
+distname            ${github.version}
+
+license             AGPL-3
+categories          devel
+maintainers         {@crowell crowell.biz:jeff} openmaintainer
+description         A modern linker.
+long_description    A modern linker.
```

You can do something here like:

```suggestion
long_description    {*}${description}
```